fusion_docs/faq.mdx

```diff
@@ -2,53 +2,67 @@
 title: FAQ
 ---
 
-# Frequently Asked Questions
+### Which cloud object stores does Fusion support?
 
-## Which object storage are supported by Fusion
+Fusion supports AWS S3, Azure Blob, and Google Cloud Storage.
```

Comment:
We also support any local storage the supports the S3 API, do we want to call this out explicitly?